js/job.jsx

```diff
@@ -431,43 +436,48 @@ class BoxPlot extends React.Component {
     return false;
   }
 
-  componentDidMount() {
-    boxplot(this.props.data, ReactDOM.findDOMNode(this), this.props.tickFormat);
+  render() {
+    return <div />;
   }
 }
 
 function boxplot(data, node, tickFormat) {
-  var margin = {top: 10, right: 100, bottom: 20, left: 100},
-      width = 220 - margin.left - margin.right,
-      height = 400 - margin.top - margin.bottom;
+  const margin = {
+    top: 10,
+    right: 100,
+    bottom: 20,
+    left: 100,
+  };
+  const width = 220 - margin.left - margin.right;
+  const height = 400 - margin.top - margin.bottom;
 
   var chart = d3.box()
-      .whiskers(iqr(1.5))
-      .width(width)
-      .height(height)
-      .tickFormat(tickFormat);
+    .whiskers(iqr(1.5))
+    .width(width)
+    .height(height)
+    .tickFormat(tickFormat);
 
   chart.domain(d3.extent(data));
 
-  var svg = d3.select(node).selectAll('svg')
-      .data([data])
+  d3.select(node).selectAll('svg')
+    .data([data])
     .enter().append('svg')
-      .attr('class', 'box')
-      .attr('width', width + margin.left + margin.right)
-      .attr('height', height + margin.bottom + margin.top)
+    .attr('class', 'box')
+    .attr('width', width + margin.left + margin.right)
+    .attr('height', height + margin.bottom + margin.top)
     .append('g')
-      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')')
-      .call(chart);
+    .attr('transform', `translate(${margin.left},${margin.top})`)
+    .call(chart);
 
   function iqr(k) {
-    return function(d, i) {
-      var q1 = d.quartiles[0],
-          q3 = d.quartiles[2],
-          iqr = (q3 - q1) * k,
-          i = -1,
-          j = d.length;
-      while (d[++i] < q1 - iqr);
-      while (d[--j] > q3 + iqr);
+    return function(d) {
+      const q1 = d.quartiles[0];
+      const q3 = d.quartiles[2];
+      const iqrange = (q3 - q1) * k;
+      let i = -1;
+      let j = d.length;
+      while (d[i += 1] < q1 - iqrange);
```

Comment:
Oh interesting...I would've thought the linter would prefer `d[++i]` over `d[i += 1]` here

Reply:
likewise…
a better fix is probably a do…while loop but this at least unblocks enforcement